.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -113,6 +113,10 @@ jobs:
     runs-on: ${{ matrix.os }}
     name: ${{ matrix.name }}
     env:
+        # With macOS we need
+        # - std::filesystem::path from 10.15
+        # - std::shared_mutex from 10.12
+        MACOSX_DEPLOYMENT_TARGET: 10.15
```

Comment:
wonder if we just always set this in CI

As the minimum version for rustc will be 10.12 but we need 10.15

https://blog.rust-lang.org/2023/09/25/Increasing-Apple-Version-Requirements.html

Reply:
Setting this on CI is fine, but it doesn't solve the problem of new versions of cc breaking the build when downstream users haven't set the environment variable.